utility/doc/E2E_FEATURE_PATH_TEMPLATE.md

```diff
@@ -186,6 +196,19 @@ This is the 🥩 (or 🥙) of the whole feature. In several PRs, you will implem
 - Additions / modifications to the [E2E testing framework](../../e2e)
 - Additions / modifications to the documentation, code structure and diagrams
 
+### Deliverables
+
+1. [The POC PR](https://github.com/pokt-network/pocket/blob/main/utility/doc/E2E_FEATURE_PATH_TEMPLATE.md#poc-proof-of-concept).
+
+   - This will not be merged, but used to A) get a review on the approach and B) breakup the work into a list of PRs
+   - Should include the bare minimum E2E test(s) to test/demo the assigned feature: [a `.feature` file](https://github.com/pokt-network/pocket/pull/869/files#diff-30c6c02e8594cf72662ab975e75a810a5bbd702f274e2eaf160c97ec14f5e642) and [the updated `.go` steps file](https://github.com/pokt-network/pocket/pull/869/files#diff-01dec4121ae8acb7a1f4bb72a6c2104827d2c2d2197eb5f45fa5c032ffba32cd)
+
+2. [The list of PRs needed for making the E2E test pass](https://github.com/pokt-network/pocket/pull/869#issuecomment-1618484939)
+
+3. PRs matching the above list
+
+   - Each PR should be about a single change and the corresponding tests
```

Comment:
Do you think point (3) is better suited for our [github issue](https://github.com/pokt-network/pocket/blob/main/.github/ISSUE_TEMPLATE/issue.md) or [github pr](https://github.com/pokt-network/pocket/blob/main/.github/PULL_REQUEST_TEMPLATE.md) template?

![Screenshot 2023-07-03 at 6 57 14 PM](https://github.com/pokt-network/pocket/assets/1892194/9c2bfa60-da00-4c18-bcad-21f1537dfd1a)

Reply:
Thanks, updated the PR template with the proposed change.